schema_registry/client/client.py

```diff
@@ -1239,10 +1249,11 @@ async def test_compatibility(
         subject: str,
         schema: typing.Union[BaseSchema, str, typing.Dict[str, typing.Any]],
         version: typing.Union[int, str] = "latest",
+        verbose: bool = False,
         headers: typing.Optional[typing.Dict] = None,
         timeout: typing.Union[TimeoutTypes, UseClientDefault] = USE_CLIENT_DEFAULT,
         schema_type: typing.Literal["AVRO", "JSON"] = utils.AVRO_SCHEMA_TYPE,
-    ) -> bool:
+    ) -> typing.Union[bool, typing.Dict[str, typing.Any]]::
```

Comment:
Extra `:`

Reply:
Fixed
You can now retrigger workflow plz